### PR TITLE
Feature/clinvar ref

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:

- Updates clinvar references

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: templates/mip_download_rd_dna_config_-1.0-.yaml [hunk 1]
```diff
@@ -339,16 +330,16 @@ reference_feature:
...
         url_prefix: ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh37/archive_2.0/2020/
+      20210415:
+        file: clinvar_20210415.vcf.gz
+        file_check: clinvar_20210415.vcf.gz.md5
+        file_index: clinvar_20210415.vcf.gz.tbi
+        outfile: grch37_clinvar_-20210415-.vcf.gz
+        outfile_check: grch37_clinvar_-20210415-.vcf.gz.md5
+        outfile_index: grch37_clinvar_-20210415-.vcf.gz.tbi
+        outfile_check_method: md5sum
+        url_prefix: https://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh37/weekly/
```
Comment (henrikstranneheim) [inline templates/mip_download_rd_dna_config_-1.0-.yaml:341]: Is this a stable URL prefix? Will this not be broken next week?

File: templates/mip_download_rd_dna_config_-1.0-.yaml [hunk 2]
```diff
@@ -358,6 +349,15 @@ reference_feature:
...
         url_prefix: ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh38/archive_2.0/2020/
+      20210415:
+        file: clinvar_20210415.vcf.gz
+        file_check: clinvar_20210415.vcf.gz.md5
+        file_index: clinvar_20210415.vcf.gz.tbi
+        outfile: grch38_clinvar_-20210415-.vcf.gz
+        outfile_check: grch38_clinvar_-20210415-.vcf.gz.md5
+        outfile_index: grch38_clinvar_-20210415-.vcf.gz.tbi
+        outfile_check_method: md5sum
+        url_prefix: https://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh38/weekly/
```
Comment (henrikstranneheim) [inline templates/mip_download_rd_dna_config_-1.0-.yaml:360]: Is this a stable URL prefix? Will this not be broken next week?

File: templates/mip_download_rd_dna_config_-1.0-.yaml (hunk 1 above)
Comment (jemten) [reply templates/mip_download_rd_dna_config_-1.0-.yaml:341]: I think so. The weekly folder, https://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh37/weekly/ seems to contain all versions

Comment (henrikstranneheim) [reply templates/mip_download_rd_dna_config_-1.0-.yaml:341]: 👍

**Review by henrikstranneheim — APPROVED** [review]
👍

**Pull request merged**